charts/prometheus-elasticsearch-exporter/values.yaml

```
@@ -18,13 +18,25 @@ image:
   pullPolicy: IfNotPresent
   pullSecret: ""
 
-## Set enabled to false if you don't want securityContext
-## in your Deployment.
-## The below values are the default for kubernetes.
-## Openshift won't deploy with runAsUser: 1000 without additional permissions.
-securityContext:
-  enabled: true  # Should be set to false when running on OpenShift
+## Set to `null` to keep pod level securityContext unspecified
+#podSecurityContext: null
+podSecurityContext:
```

please use it like this to use current default.
```suggestion
podSecurityContext: {}
```

---

but curretnly podSecurityContext is not used and so you would keep the defaults and still have it configurable.